instrumentation/opentelemetry-instrumentation-sqlalchemy/src/opentelemetry/instrumentation/sqlalchemy/__init__.py

```diff
@@ -224,6 +231,7 @@ def _instrument(self, **kwargs):
     def _uninstrument(self, **kwargs):
         unwrap(sqlalchemy, "create_engine")
         unwrap(sqlalchemy.engine, "create_engine")
+        unwrap(sqlalchemy.engine.create, "create_engine")
```

Comment:
Should we move this to the if statement `parse_version(sqlalchemy.__version__).release >= (1, 4)` as well?

Reply:
Agreed!